client/package.json

```diff
@@ -13,6 +13,7 @@
     "@fortawesome/react-fontawesome": "^0.2.0",
     "@mui/icons-material": "^5.10.9",
     "@mui/material": "^5.10.9",
+    "@mui/styled-engine-sc": "^5.10.6",
```

Comment:
why is this needed

Reply:
It's designed to use styled-components as the main styled engine instead of @emotion/styled.